src/org/javarosa/core/services/locale/LocalizationUtils.java

```diff
@@ -84,7 +87,7 @@ private static void parseAndAdd(OrderedMap<String, String> locale, String line,
                 if(line.trim().equals("")) {
                     //Empty Line
                 } else {
-                    Std.out.println("Invalid line (#" + curline + ") read: " + line);
+                    logger.info("Invalid line (#{}) read: {}", curline, line);
```

Comment:
I assume you weren’t able to test every one of these changes where you added the `{}` placeholders (a change I like), so I’ll just mention the risk of breakage.

Reply:
Thanks for noticing :)

I reviewed all the changes trice, but some problem might have slipped... I'll update the issue description to reflect this.